examples/images/diffusion/test_ci.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euxo pipefail
+
+conda env create -f environment.yaml
+
+conda activate ldm
+
+conda install pytorch==1.12.1 torchvision==0.13.1 torchaudio==0.12.1 cudatoolkit=11.3 -c pytorch
+pip install transformers diffusers invisible-watermark
+
+CUDA_EXT=1  pip install colossalai
```

Comment:
I believe the ci docker has already installed colossalai, transformers and so on.

Reply:
> I believe the ci docker has already installed colossalai, transformers and so on.

a new conda environment has been created, so it needs to install the latest colossalai